packages/core/integration-tests/test/scope-hoisting.js

```diff
@@ -3611,7 +3611,8 @@ describe('scope hoisting', function () {
       assert.equal(output, 42);
     });
 
-    it('builds commonjs modules that assigns to module.exports before exports', async function () {
+    // TODO this should be working
+    it.skip('builds commonjs modules that assigns to module.exports before exports', async function () {
```

Comment:
The output right now is
```js
parcelRequire.register("5CKoA", function(module, exports) {
module.exports = 42;
module.exports.foo = 27;

});
```
which is invalid in strict mode (something like "cannot add property to number")

Reply:
Ah, forgot to include the input code:
```
module.exports = 42;
exports.foo = 27;
```
so this really is a failing test (just like the other one with the undeclared variable).
(The correct behaviour here is that "42" is exported and the second line does nothing)